code-challenges.test.js

```diff
@@ -22,6 +35,38 @@ const fibLength2 = 10
 
 // b) Create the function that makes the test pass.
 
+// Create a function called fib
+// input: integer for number of terms
+// if input is less than or equal to 2 then return error
+// create a variable to hold first number ei 1
+// create a variable to hold second number ei 1
+// declare a variable to hole the next term
+// create an array to hold the result
+// use a for loop that starts at 1, runs until the number of iterations is equal to the input, add one to iterator
+// push the first number to result
+// assign next term to the first num plus the second num
+// set the first number to the second number
+// set the second number to the first number plus the second number
+// return the result array after completion of iterations
+
+const fib = (numOfTerms) => {
+  if (numOfTerms <= 2) {
+    return "numOfTerms must be greater than 2"
```

Comment:
Did you want to use strong interpolation here? Or just happened to use the same name?